reduceEpsilons.py

```diff
@@ -11,9 +11,9 @@
 # Add the common include directory before importing our custom libraries
 sys.path.append("include")
 
-from ascFile import *
-from calibrationLib import *
-from utility import *
+from include.ascFile import *
+from include.calibrationLib import *
+from include.utility import *
```

Comment:
Regression, but looking at the rest of the file it seems like you are running with an out of date copy, which would explain the rest of the regressions.